.github/workflows/rust.yml

```diff
@@ -104,8 +104,26 @@ jobs:
       if: ${{ github.ref != 'refs/heads/main' }}
     - name: Report cargo version
       run: cargo --version
-    - name: Report disk space
-      run: df -h .
+    - name: Remove unnecessary software
```

Comment:
FWIW this feels like a gross hack; I will not be offended if you say so and don't want to merge this!